packages/amplify-codegen/src/commands/models.js

```diff
@@ -105,7 +105,9 @@ function getModelOutputPath(context) {
     case 'javascript':
       return 'src';
     case 'android':
-      return 'app/src/main/java/';
+      return projectConfig.android && projectConfig.android.config && projectConfig.android.config.ResDir
+        ? path.join(projectConfig.android.config.ResDir, '..', 'java')
+        : 'app/src/main/java/';
```

Comment:
Should we use path.join here instead of forward slashes for windows compatibility?